lib/fluent/command/config_validator.rb

```
@@ -0,0 +1,88 @@
+          end
+          
+          if valid
+            puts "Configuration is valid."
+          else
+            puts "Configuration is invalid."
+          end
+          puts
+        end
+        
+        exit exit_code
+      end
```

RuboCop (1.75.5)

[convention] Assignment Branch Condition size for run is too high. [<8, 29, 10> 31.7/23]

(Metrics/AbcSize)